src/shared/util.js

```diff
@@ -288,27 +288,9 @@ function deprecated(details) {
   console.log('Deprecated API usage: ' + details);
 }
 
-// Fatal errors that should trigger the fallback UI and halt execution by
-// throwing an exception.
-function error(msg) {
-  if (verbosity >= VERBOSITY_LEVELS.errors) {
```

Comment:
Do we still use the verbosity anywhere now? If not, we should remove all traces of it as well.